### PR TITLE
Back-fill changed classes on relevant components

### PR DESCRIPTION
**Pull request by misteroneill**

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -41,6 +55,19 @@
     }
   }
 
+  Object.keys(backfillClasses).forEach(function(className) {
+    backfillClasses[className].forEach(function(name) {
+      var Component = videojs.getComponent(name);
+      var createEl = Component.prototype.createEl;
+
+      Component.prototype.createEl = function() {
```
Comment (dmlap) [inline src/plugin.js:63]: Is overriding createEl necessary for this? What about just adding the classes post-player init?

**Comment by gkatsev** [comment]
LGTM

File: src/plugin.js (hunk 1 above)
Comment (gkatsev) [inline src/plugin.js:63]: Doing it this way will also likely reduce reflows

Comment (misteroneill) [inline src/plugin.js:63]: There's no guarantee that all components are in place on player init. For instance, what if they get generated by a plugin at some later point? This approach seems way more foolproof.

Comment (dmlap) [inline src/plugin.js:63]: If you guys both feel this is the right approach, it's cool with me. It's a pretty fundamental method to monkey-patch though.

**Pull request merged**